src/lib/key-binder.c

```diff
@@ -173,18 +173,21 @@ static GdkFilterReturn brisk_key_binder_filter(GdkXEvent *xevent, GdkEvent *even
                     !self->wait_for_release) {
                         if (mods == binding->mods) {
                                 self->wait_for_release = TRUE;
+                                XAllowEvents(display, AsyncKeyboard, xev->xkey.time);
                         }
                 } else if (xev->xkey.keycode == binding->keycode && self->wait_for_release) {
                         /* capture release within same shortcut sequence */
                         if (xev->type == KeyRelease) {
                                 self->wait_for_release = FALSE;
                                 binding->func(event, binding->udata);
                         }
+                        XAllowEvents(display, AsyncKeyboard, xev->xkey.time);
                 } else {
                         /* when breaking the shortcut sequence, send the event up the window
-                         * hierarchy
-                         * in case it's part of a different shortcut sequence (e.g. <Mod4>a) */
-                        XSendEvent(display, (Window)NULL, TRUE, KeyPressMask | KeyReleaseMask, xev);
+                         * hierarchy in case it's part of a different shortcut sequence
+                         * (e.g. <Mod4>a) */
+                        XSendEvent(display, GDK_WINDOW_XID(self->root_window), TRUE, KeyPressMask | KeyReleaseMask, xev);
```

Comment:
This is the actual fix.

All instances of `XAllowEvents` are there to ensure that brisk-menu itself, if it ever finds itself locking on key events, can release them from the queue. It's an extremely low probability scenario, given that we grab the keys asynchronously, but they have no effect if there's no lock, so I figured we might as well have them there just in case (it can get pretty ugly very quickly if brisk locks the keyboard without releasing - you essentially need to exit the desktop environment and kill the offending process)